### DISCUSSION TITLE
How to smooth scroll top after submit or navigate between page?

### DISCUSSION THREAD

**Discussion by vanthao03596** [orchidsoftware/platform#1732] [Q&A]
I'm using Turbo in my application. I can't find the way orchid do smooth scroll top when submit form or navigate.
Someone can help me ?

**Comment by didix16** [comment]
Hi , I'm wondering too. BTW did you solve it or just waiting a solution?

**Reply by vanthao03596** [reply]
@didix16  i'm still waiting for solution.

**Reply by tabuna** [reply]
I didn't do anything specifically for this

**Reply by didix16** [reply]
Ok thanks anyway. I've been searching in Turbo repository and it seens there are a lot of people having problems with scrolling after form submission or back click. These are just a few links related with scroll issues.

* https://github.com/hotwired/turbo/issues/171
* https://github.com/hotwired/turbo/pull/79
* https://github.com/turbolinks/turbolinks/pull/420
* https://github.com/turbolinks/turbolinks/issues/333
* https://github.com/turbolinks/turbolinks/issues/248
